plugins/cisco_firepower_management_center/help.md

```diff
@@ -706,26 +706,18 @@ Example output:
 
 
 ## Troubleshooting
-  
-* The certificate authentication is used solely in two actions, namely: `Add Scan Result` and `Bulk Add Scan Result`. Please remember, that the data entered into the certificate input field in the connection is required to be a base-64 encoded PKCS12 certificate file, exported from the Firepower Management Center server. The passphrase is a password created during the PKCS12 certificate file export.
-* `SSL Verify` field is used by all other actions (i.e. excluding `Add Scan Result` and `Bulk Add Scan Result`), for SSL certificate verification. If the certificate is self-signed then SSL Verify should be set to `False` for those actions.
+
+The certificate authentication is used solely in two actions, namely: `Add Scan Result` and `Bulk Add Scan Result`. Please remember, that the data entered into the certificate input field in the connection is required to be a base-64 encoded PKCS12 certificate file, exported from the Firepower Management Center server. The passphrase is a password created during the PKCS12 certificate file export.`SSL Verify` field is used by all other actions (i.e. excluding `Add Scan Result` and `Bulk Add Scan Result`), for SSL certificate verification. If the certificate is self-signed then SSL Verify should be set to `False` for those actions.
 
 # Version History
 
-* 2.1.2 - `Block URL Policy` - Refactor the action to use local API calls | Remove vulnerable dependencies
-* 2.1.1 - Updated to latest SDK version | Fixed issue related to pagination
-* 2.1.0 - `Check if Address in Group`: Extended search for manually added literals | Added new output field `literal_objects`
-* 2.0.1 - Fix issue in Add Address to Group action where Network Groups that had no objects would result in action failure
-* 2.0.0 - Combine Cisco Firepower and Cisco Firepower Management Center plugins
-* 1.2.0 - New actions - Check If Address in Group, Add Address to Group, Remove Address from Group
-* 1.1.0 - New actions - Create Address Object, Delete Address Object
-* 1.0.1 - New spec and help.md format for the Extension Library
-* 1.0.0 - Initial plugin
+* 2.1.3 - Bumped 'cryptography' | SDK Bump to 6.1.2
```

Comment:
Missing some version_history here in the plugin.spec.yaml

Reply:
Well spotted! Don't know how I managed to miss that 🤨